apm-collector/apm-collector-storage/collector-storage-es-provider/src/main/java/org/apache/skywalking/apm/collector/storage/es/dao/ui/SegmentDurationEsUIDAO.java

```diff
@@ -57,8 +58,9 @@ public TraceBrief loadTop(long startSecondTimeBucket, long endSecondTimeBucket,
         List<QueryBuilder> mustQueryList = boolQueryBuilder.must();
 
         if (startSecondTimeBucket != 0 && endSecondTimeBucket != 0) {
-            //TODO second
-            mustQueryList.add(QueryBuilders.rangeQuery(SegmentDurationTable.COLUMN_TIME_BUCKET).gte(startSecondTimeBucket).lte(endSecondTimeBucket));
+            mustQueryList.add(QueryBuilders.rangeQuery(SegmentDurationTable.COLUMN_TIME_BUCKET)
+                    .gte(TimeBucketUtils.INSTANCE.secondToMinute(startSecondTimeBucket))
```

Comment:
This time format should already done in `TraceQuery#queryBasicTraces#L71`. And the query works in our demo env and product env.